Feature branch

HeyItsGwen opened the pull request:
pull request for project review

HeyItsGwen committed:
Add file skeletons

Added the skeletons of the basic layouts to the empty html/css files. (nav, main, footer, as well as

aracelicaes committed:
added index info

added organized index elements. Will fill in with info little by little. Added notes for us to know what to fill it in with.

HeyItsGwen committed:
Worked on nav bar

added skeleton and sytling to nav bar to replicate the article website. Added empty images, links, and button for nav.

HeyItsGwen committed:
positioned and styled navbar

Positiond the different elements within the navbar and added the bottom border to it. Styled the Log in button to look like the reference

aracelicaes committed:
Created images folder

Uploading all the images

aracelicaes committed:
Inserted images and Header

Inserted images, added id's, classes, alts. Also inserted header information, h1, spans, figures and figcaptions.

aracelicaes committed:
Trying to style title

Started styling Title.

HeyItsGwen committed:
Style title and nav

Styled the nav text to match the reference, styled the article title, and centered it (kind of) to match the reference. Centered the figure with the first image.

HeyItsGwen committed:
Styled fig caption, article

Added styling and proportions to the fig caption, article info section, and article itself. Added some text for the article, as well as subheadings.

aracelicaes committed:
Lined Icons, Date, Author

Fixed HTML for top icons.; Displayed icons, author and date in a row.

HeyItsGwen committed:
fix icon and date style

Fixed icon and date styling to look more like the reference. Added positioning and flex margins to icons.1

aracelicaes committed:
Added paragraph content, styled top icons

Added the paragraph contents.
Added the section 2 paragraph contents and elements.
Added an image for the theory of Inflation
Added H2 title for section 2 along with a figure and figcaption (top and bottom)

Styled topIcons towards the right side and Date towards the left using relative.

HeyItsGwen committed:
added images

added images and came to a revelation about how the site resizes

HeyItsGwen committed:
finished the skeleton

added paragraphs, sections, headers and lists to make the page match the reference. styled different sections to move where they're supposed to on the page.

HeyItsGwen committed:
Added content

Added content to the article: text and youtube video.

aracelicaes committed:
Started creating image section

Started adding image section for articles.

aracelicaes committed:
Merge branch 'Feature-Branch' of https://github.com/HeyItsGwen/NTY-Clone-Ara-and-Gwen into Feature-Branch

aracelicaes committed:
Edited HTML, added some CSS

The Theory Of Inflation, Bottom Icons, Read More Section, Added Comments, Added Images, Added some of the List Bottom Nav Menu Items
Added CSS stylings for the bottom Read More social media Icons and button. They have same styling as icons on top of page.

HeyItsGwen committed:
Added content, styled footer

Added content to the bottom info box, styled links, added the legal info to the footer, made resizeable, and styled the links to not look like garbage.

aracelicaes committed:
Added Stylings to The Theory of Inflation

Styled the text and image of The Theory of Inflation Section, also adjustied px of top for the next sections below.

HeyItsGwen committed:
add stickler stuff

Added stylelint files and stickler files for repo

HeyItsGwen committed:
re-add linters

change linter files

HeyItsGwen committed:
remove linters

still trying to make them work

HeyItsGwen committed:
add linters

adding linters for stickler

HeyItsGwen committed:
fix some linting issues

added space before brackets and changed line spacing for comments

aracelicaes committed:
HTML Fix Social Media Icons, Styled Read More Section

Added HREFs to the Social Media Links, Also began styling the REad More Section, images and text.

aracelicaes committed:
Finished Styling Read More Section

Finished Styling Read More Section

HeyItsGwen committed:
Style footer and title

Styled the footer lists, added images, aligned the title and fixed the validation issues with it.

HeyItsGwen committed:
make images responsive

add width values to the containers and images within them to make them responsive on screen resize.

aracelicaes committed:
Created Figures for SM Icons, added hover

Created figures with gray background for social media icons. Added hover options to be white.

aracelicaes committed:
Validation Correction HTML

Fixed HTML Validation error. Added h2 to sections

aracelicaes committed:
Validateed and Fixed all CSS and HTML errors

Files have validated 100%.

HeyItsGwen committed:
remove css reference file

whoops

HeyItsGwen committed:
fixing linter issues

changing hex codes and spaces and such

HeyItsGwen committed:
More linter issues

you know the deal it's stuff like spaces and that you know come on

HeyItsGwen committed:
more linter errors

leading zeroes, spaces, and commas

HeyItsGwen committed:
linter error fixes

descending specificity and comma list separation

HeyItsGwen committed:
fix linter errors

The last ones I think, more comma list things and descending specificity

HeyItsGwen committed:
updated readme

updated the readme for the feature branch. added a screenshot

HeyItsGwen committed:
update README again

added more description to the README and tried to fix the screenshot

KaushikShivam commented:
Pull Request under the review by Shivam Kaushik

KaushikShivam reviewed (changes requested):
# Status: Changes Requires 🔴 

Hello, @HeyItsGwen & @aracelicaes 👋 
Great work on the project so far!! The code looks professional and I like that you guys have used proper semantic tags and have proper comments throughout your code. Great job!! 👏 

However, I need you guys to make a couple of fixes in order to pass this review.

## Mandatory changes
1. Navigation bar
    - [x] The NY Times logo should be a link.
2. ```Read more from science``` section
     - [x] It contains links to other articles with image previews
3. Footer
    - [x] There should be a fixed footer that is stuck to the bottom of the page
    - [x] You need to add the markup and styles for it. 
![Screen Shot 2019-10-26 at 3 45 58 PM](https://user-images.githubusercontent.com/14944115/67618074-b9f7e000-f807-11e9-8d23-91f69591eba7.png)

4. Other implementation issues
    - [x] As this is a project aimed to practise positioning and floats, USE floats to align elements instead of flexbox. You will have many chances to practise flexbox later in the course.

 ### That's all, guys! 
**Make these changes and submit for a code review, again**
You guys are doing exceptionally well. I'm sure these changes won't take you much time. 👍 

**If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.**

Happy coding! 😄 
[Shivam Kaushik](https://github.com/KaushikShivam)
[Github](https://github.com/KaushikShivam) [Twitter](https://twitter.com/kShivamDev) [Linkedin](https://www.linkedin.com/in/shivam-kaushik-bb8162102/)

HeyItsGwen committed:
fix errors from review

Made the NYT logo's links, made the readmore section images/description links, reorganized bottom section and footer.

HeyItsGwen committed:
fix linter errors

fixed a few descending specificity errors and a space error

HeyItsGwen committed:
fix linter error

removed an extra empty line

HeyItsGwen committed:
Add footer skeleton

Added skeleton for the sticky footer part

aracelicaes committed:
Changed flex to float

Changed the CSS from flex to float

aracelicaes committed:
Finished the Sticky Foother

styled and positioned the sticky footer

HeyItsGwen committed:
Fix validation issues

Fixed w3 validation issues in html and css

HeyItsGwen committed:
fix linter issues

changing order of selectors

ebukaume commented:
## Approved! :champagne: 

Hi @HeyItsGwen  and @aracelicaes :wave: 

**You guys have done well. You are approved to merge this branch.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

HeyItsGwen committed:
Merge branch 'master' into Feature-Branch
